modules/reitit-core/project.clj

```diff
@@ -0,0 +1,7 @@
+(load-file "../../.deps-versions.clj")
```

Comment:
I hate it that Clojure project files have to be evaluated as code to be parsed. This is a language that promotos data-driven interfaces, after all!

Reply:
good point. Maven has declarative includes, maybe leiningen should too? 

```xml
  <parent>
    <groupId>org.codehaus.mojo</groupId>
    <artifactId>my-parent</artifactId>
    <version>2.0</version>
    <relativePath>../my-parent</relativePath>
  </parent>
```

and actually, it does have this: https://github.com/achin/lein-parent